docs/protocol/vapp/examples/poker.md

```
@@ -0,0 +1,124 @@
+    betting_round: "preflop" | "flop" | "turn" | "river" | "showdown",
+    current_turn: uint
+}
+```
+
+This is the key V2 advantage: ModuleState holds the full game state without
+participants needing to sign it. Players sign only their action. The module
+computes everything else deterministically.
+
+## Events
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)